### DISCUSSION TITLE
Disable blank space in DAX formulas before and after parentheses

### DISCUSSION THREAD

**Discussion by lukiz84** [TabularEditor/TabularEditor3#981] [🙋♀️TE3 Q&A] [answered]
Hi,

in the past, the dax formatter didn't add the blanks after opening parentheses and before closing parentheses. How can I disable this?

What I don't want:

VAR Xyz = CALCULATE( measure1, Date[Year] = 2022 )

What I want:

VAR Xyz = CALCULATE(measure1, Date[Year] = 2022)

BR
Lukas

**Comment by otykier** [comment, marked as answer]
Hi Lukas,

This is configurable. You’ll find the settings under **Tools > Preferences > DAX Editor > Auto Formatting**.

Look for the **Pad parentheses** setting and turn it *off*.

**Comment by lukiz84** [comment]
Ah thanks, sometimes I'm a little blind :)
